std/container/array.d

```diff
@@ -539,11 +525,13 @@ Complexity: $(BIGOH 1)
     }
 
 /**
-Ensures sufficient capacity to accommodate $(D e) elements.
+Ensures sufficient capacity to accommodate $(D e) _elements.
+If $(D e < capacity), this method will not decrease the capacity.
 
 Postcondition: $(D capacity >= e)
 
-Complexity: $(BIGOH 1)
+Complexity: $(BIGOH e) if $(D e > capacity),
```

Comment:
> Unless I'm missing something, that doesn't happen right now?

One more reason to avoid std.container for the time being ...

On a serious note - this is a disaster, any half-sensible array implementation preallocates spare capacity in event of grow.